storage/table_batch.go

```diff
@@ -131,14 +131,32 @@ func (t *TableBatch) MergeEntity(entity *Entity) {
 // the changesets.
 // As per document https://docs.microsoft.com/en-us/rest/api/storageservices/fileservices/performing-entity-group-transactions
 func (t *TableBatch) ExecuteBatch() error {
-	changesetBoundary := fmt.Sprintf("changeset_%s", uuid.NewV1())
+
+	// Using `github.com/marstr/guid` is in response to issue #947 (https://github.com/Azure/azure-sdk-for-go/issues/947).
```

Comment:
I think this comment block should go in the change description and not here.

Reply:
@marstr  I would suggest to create an issue to track that fact the we need to change this once satori releases v2

Reply:
Issue added.

Reply:
I moved the bulk of the comment to address your note, @jhendrixMSFT  :)